upup/pkg/fi/cloudup/awstasks/autoscalinggroup.go

```diff
@@ -264,6 +276,14 @@ func (v *AutoscalingGroup) RenderAWS(t *awsup.AWSAPITarget, a, e, changes *Autos
 			VPCZoneIdentifier:    fi.String(strings.Join(e.AutoscalingGroupSubnets(), ",")),
 		}
 
+		for _, k := range e.LoadBalancers {
```

Comment:
I think we'd want to sort both of these lists (and perhaps in `pkg/model/awsmodel/autoscalinggroup.go` too) so that a difference in ordering wont report changes and to ensure the terraform files are generated consistently.

Reply:
Added SortLiterals for the TG and SG list. 

I don't believe `pkg/model/awsmodel/autoscalinggroup.go` needs to be sorted since the AWS API doesn't care about ordering (If it did we'd be seen a ton of unintended resources updates as nothing under `pkg/model/awsmodel/` is sorted.).